### PR TITLE
Set system time without isys

### PR DESCRIPTION
**Pull request by VladimirSlavik**
This changes the time setting mechanism and consequently removes the `isys` module.

Depends on https://github.com/rhinstaller/anaconda/pull/4358 and https://github.com/weldr/lorax/pull/1271 = blocked, needs rebase on top of that.

Kickstart tests also depend on the other PRs.

### CODE REVIEW

**Review by M4rtinK — APPROVED** [review]
Looks good to me! A really nice cleanup + huge thanks! :D

File: pyanaconda/timezone.py [hunk 1]
```diff
@@ -152,3 +154,42 @@ def get_timezone(timezone):
...
+    day = day if day is not None else now.day
+    hour = hour if hour is not None else now.hour
+    minute = minute if minute is not None else now.minute
+    second = now.second
+
+    set_date = datetime.datetime(year, month, day, hour, minute, second, tzinfo=tz)
+    epoch_seconds = int(set_date.timestamp())
+
+    log.info("Setting system time to %s UTC", time.asctime(time.gmtime(epoch_seconds)))
+    execWithRedirect("date", ['--set=@{}'.format(epoch_seconds)])
```
Comment (poncovka) [inline pyanaconda/timezone.py:195]: Do we have in dependencies whatever provides the `date` tool?

**Review by poncovka — APPROVED** [review]
Thank you!

File: pyanaconda/timezone.py (hunk 1 above)
Comment (VladimirSlavik) [reply pyanaconda/timezone.py:195]: Good point... I will try to find that.

**Pull request merged**